lib/ansible/modules/network/nxos/nxos_hsrp.py

```diff
@@ -272,8 +249,9 @@ def get_hsrp_group(group, interface, module):
 
 def get_commands_remove_hsrp(group, interface):
     commands = []
-    commands.append('interface {0}'.format(interface))
-    commands.append('no hsrp {0}'.format(group))
+    cmds = 'interface {0}'.format(interface), 'no hsrp {0}'.format(group)
```

Comment:
Updated the change with lambda https://github.com/ansible/ansible/pull/24770/commits/f0c74e3a8f1bb368a8fdd3383a07d672ec04da37.